modules/ossm-architecture.adoc

```diff
@@ -7,15 +7,12 @@
 
 {ProductName} is logically split into a data plane and a control plane:
 
-The *data plane* is a set of intelligent proxies deployed as sidecars. These proxies intercept and control all inbound and outbound network communication between microservices in the service mesh. Sidecar proxies also communicate with Mixer, the general-purpose policy and telemetry hub.
+The *data plane* is a set of intelligent proxies deployed as sidecars. These proxies intercept and control all inbound and outbound network communication between microservices in the service mesh. 
 
-* *Envoy proxy* intercepts all inbound and outbound traffic for all services in the service mesh. Envoy is deployed as a sidecar to the relevant service in the same pod.
+The architectural units used by previous versions have been replaced by Istiod. In 2.0 the control plane components Mixer, Pilot, Citadel, Galley and the sidecar injector functionality have been combined into a single component, Istiod.
```

Comment:
Somewhere we lost the data plane/control plane split in this topic.  Istiod is the Control Plane component.

https://istio.io/latest/docs/concepts/what-is-istio/#why-use-istio

Also, I don't think we should be talking about previous versions anywhere but the release notes and upgrade topics.  This topic should be explaining the current state for new users.